server/models/index.js

```diff
@@ -0,0 +1,54 @@
+//both: name, email, password, rating, city
+//customer: jobs: name, description, required skills, 
+//need date, city, job by id, proposals by id
+//contractor: skills, jobs completed.  Proposals: contractor
+//name, cost estimate, start date estimate, time frame estimate
+
+
+
```

Comment:
Suggest using 'user/User' in place of 'base/Base' to be consistent with file name and to make context clear. 
Change collection to 'users', this is referencing the collection under which users are stored. handymandb is the database name.